fw/http.c

```diff
@@ -1658,14 +1658,10 @@ tfw_http_hm_control(TfwHttpResp *resp)
 	if (tfw_http_hm_suspend(resp, srv))
 		return;
 
-	if (!tfw_srv_suspended(srv) ||
-	    !tfw_apm_hm_srv_alive(resp->status, &resp->body, resp->msg.skb_head,
-				  srv->apmref))
-	{
-		return;
-	}
-
+	if (tfw_srv_suspended(srv) && tfw_apm_hm_srv_alive(resp, srv)) {
```

Comment:
You've rewritten this condition, but are sure that logic is correct? The main go of the issue is find out whether should we do it this way or not.

Let's see, now you calling `tfw_apm_hm_srv_alive()` only for suspended servers(the same as before), but `tfw_http_hm_control()` calls only when response received, it means we can receive response from suspended server. Why?

Reply:
!a || !b = !(a && b)